src/App.jsx

```diff
@@ -103,6 +138,60 @@ function App() {
             />
```

Comment:
⚖️ **Midlevel Developer**

Nitpick: The new input fields for Social Security and Real Rate of Return (App.jsx lines 138-138, 216-216) follow the same style as previous inputs, but the placeholder values are inconsistent. For example, the inflation rate field says "e.g., 3" (which is a percentage), but Social Security uses "e.g., 2500" (which is dollars). Can we standardize the placeholder format and maybe add units to the labels for clarity?